tests/cpp_api_parser_test.py

```diff
@@ -41,11 +42,134 @@ def test_enable_if_transform():
         allow_paths=[re.escape(input_path)],
         disallow_namespaces=[re.compile("^std$")],
         compiler_flags=["-std=c++17"],
-        input_content=input_content)
+        input_content=input_content,
+    )
 
     output = api_parser.generate_output(config)
 
     entities = list(output["entities"].values())
     assert len(entities) == 1
     requires = entities[0].get("requires")
     assert requires
+
+
+class TestCommentStrip:
+    config = api_parser.Config(
+        input_path="a.cpp",
+    )
+
+    def assert_output(self, expected: str):
+        output = api_parser.generate_output(self.config)
+        doc_strings = [v["doc"]["text"] for v in output.get("entities", {}).values()]
+        assert expected in doc_strings
+
+    def test1(self):
```

Comment:
Potentially these test cases could be made into a single parameterized test case using `pytest.mark.parametrize`, where one parameter is the input source code and the other parameter is the expected docstring.

Reply:
I don't have a lot of experience with pytest. I'm actually writing unit tests with pytest right now for a different project (needed my overdue attention). So, I will look into the `mark.parameterize` idea.

Reply:
Took me a while, but I got the hang of using `pytest.mark.parametrize` ("parametrize" is kind of a weird spelling IMO). The only down side to this is that it makes failed tests harder to read (especially when parametrizing numerous args).

I also added a couple tests to check that non-docstring comments are removed.